### ISSUE TITLE
Connection doesn't work after update 

### ISSUE THREAD

**Issue by mivaken** [JoDehli/PyLoxone#183]
Last week I updated to the latest loxone config. Since the. The connection between loxone and ha isn't working anymore.I can't figure out why it isn't working any more

**Comment by beeper77** [comment]
I also get an error message after the last update. My error in the logfile is:
`2023-06-12 07:46:42.362 ERROR (MainThread) [homeassistant.config_entries] Error setting up entry PyLoxone for loxone
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/config_entries.py", line 387, in async_setup
    result = await component.async_setup_entry(hass, self)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/config/custom_components/loxone/__init__.py", line 195, in async_setup_entry
    if not await miniserver.async_setup():
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/config/custom_components/loxone/miniserver.py", line 140, in async_setup
    request_code = await self.lox_config.getJson()
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/config/custom_components/loxone/api.py", line 122, in getJson
    api_resp = await client.get("/jdev/cfg/apiKey")
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/httpx/_client.py", line 1757, in get
    return await self.request(
           ^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/httpx/_client.py", line 1530, in request
    return await self.send(request, auth=auth, follow_redirects=follow_redirects)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/httpx/_client.py", line 1617, in send
    response = await self._send_handling_auth(
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/httpx/_client.py", line 1645, in _send_handling_auth
    response = await self._send_handling_redirects(
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/httpx/_client.py", line 1703, in _send_handling_redirects
    raise exc
  File "/usr/local/lib/python3.11/site-packages/httpx/_client.py", line 1685, in _send_handling_redirects
    await hook(response)
  File "/config/custom_components/loxone/api.py", line 84, in raise_if_not_200
    raise LoxoneHTTPStatusError(
custom_components.loxone.api.LoxoneHTTPStatusError: Miniserver response code 500`

I am unsure if and update for pyloxone is needed or the miniserver has a bug.

**Comment by JoDehli** [comment]
Are you both using a gen 2 minieserver? I can not guarantee a stable connection to a gen2 as they changed again the communication. I do not have a gen2 so I can not implement it for the newest changes.

**Comment by beeper77** [comment]
Yes, I am using a gen2. I created a small test python script to see what happens in the getJson function. To simplify my test I used the ip address, port 80 (normaly using 443 as configured in the miniserver) and I get no error. So I went back to home assistant used the ip address and port 80. voila it's working again. I don't know why. Switched back to 443 and it's also working. Before I used the loxone dns name. Maybe this isn't working anymore.
